test/files/pos/t10511.check

```diff
@@ -1,9 +0,0 @@
-t10511.scala:2: warning: object DeprecatedFloatOrdering in object Ordering is deprecated (since 2.13.0): There are multiple ways to order Floats (Ordering.Float.TotalOrdering, Ordering.Float.IeeeOrdering). Specify one by using a local import, assigning an implicit val, or passing it explicitly. See the documentation for details.
```

Comment:
I don't think this file should be deleted - the test should actually spit out these errors. (If it doesn't, I believe it's because someone deleted the flags file and replaced it with an ineffective `// scalac: -deprecation` comment. I think for this one, the flags file should be restored, as the test was intended to check these deprecation warnings.

Reply:
I didn't realize by looking that it was intended. Don't impugn the tool args comment. The output is checked only if the check file exists, which is different from `neg` tests, where output is always checked and a missing file means no output. That's why checkfile wasn't supported for pos, and just documenting that caveat makes me want to disallow it again. The status quo was that if you want to check warning, you have to use `-Xfatal-warnings` with a `neg` test. It sounds like users will just be confused otherwise. The pos test would just error if there is a check file.